trinity/chains/light.py

```diff
@@ -0,0 +1,208 @@
+import asyncio
+import inspect
+from typing import (  # noqa: F401
```

Comment:
We shouldn't have to `noqa: F401` very often anymore in the `p2p` and `trinity` namespace.  Can you check if dropping this is doable?

Reply:
I didn't want to do only one file, but I also didn't want to obscure this PR. So I opened #743.